c2corg_ui/static/js/utils.js

```diff
@@ -123,6 +123,26 @@ app.utils.formatDate = function(date) {
 
 
 /**
+ * check that the start date if before end date.
+ * Return true is start > end
+ * @export
+ * @param {Object} outing
+ * @return {(boolean | undefined)}
```

Comment:
> ... but it's of no use in this particular case.

In this function you want to check if the start date is before the end data. You can answer this question with `true` or `false`. I don't see why you would need a third value `undefined`.